### PR TITLE
[debugger] Bump protocol for  multi threaded single step implementation

### PR DESCRIPTION
**Pull request by thaystg**
As suggested by @jbevain we should bump the protocol to implement multi threaded single step.

### CODE REVIEW

**Comment by thaystg** [comment]
@monojenkins build failed

File: mcs/class/Mono.Debugger.Soft/Test/dtest.cs [hunk 1]
```diff
@@ -65,9 +65,6 @@ void StackTraceDump (Event e)
 	Event GetNextEvent () {
 		var es = vm.GetNextEventSet ();
 		Assert.AreEqual (1, es.Events.Length);
-		if (step_req != null && es [0] != null && es [0].Request != null && es [0].Request is StepEventRequest && ((StepEventRequest)es [0].Request).GetId() != step_req.GetId()) {
```
Comment (vargaz) [inline mcs/class/Mono.Debugger.Soft/Test/dtest.cs:68]: Unrelated ?

Comment (thaystg) [reply mcs/class/Mono.Debugger.Soft/Test/dtest.cs:68]: I inserted this parte of the code with the multi threaded single step, but it's not necessary. So I removed with this PR that is bumping the protocol, do you think that I should do a separate one?

**Comment by thaystg** [comment]
@monojenkins backport 2020-02
